cluster/ubuntu/util.sh

```diff
@@ -457,3 +461,24 @@ function kube-push {
 function prepare-e2e() {
   echo "Ubuntu doesn't need special preparations for e2e tests" 1>&2
 }
+
+# Set proxy environment variables
+function setProxy() {
+  source "kube/${KUBE_CONFIG_FILE-"config-default.sh"}"
+  if grep '!env_reset' /etc/sudoers;then
+    echo "env_reset has been disabled"
+  else
+    sed -i -e 's|env_reset|!env_reset|' /etc/sudoers
+  fi
+  export http_proxy=${HTTP_PROXY}
+  export https_proxy=${HTTPS_PROXY}
+}
+
+# Unset proxy environment variables
+function unsetProxy() {
+  if grep '!env_reset' /etc/sudoers;then
```

Comment:
This function does not revert ```env_reset``` in ```/etc/sudoers```, i.e. if it was ```!env_reset``` before ```setProxy()```, this function will set it to ```env_reset```.